lib/client/ipam.go

```diff
@@ -513,7 +513,7 @@ func (c ipams) ReleaseHostAffinities(host *string) error {
 
 // ReleasePoolAffinities releases affinity for all blocks within
 // the specified pool across all hosts.
-func (c ipams) ReleasePoolAffinities(pool common.IPNet) error {
+func (c ipams) ReleasePoolAffinities(pool *common.IPNet) error {
```

Comment:
ReleasePoolAffinities() is part of the northbound API - so we need to decide whether a pointer to an IPNet is the right thing or just an IPNet.